src/Plugin/EntityBrowser/Display/Modal.php

```diff
@@ -177,7 +173,7 @@ public function addAjax(array &$form) {
     );
 
     $form['actions']['submit']['#ajax'] = array(
-      'callback' => array(get_class($this), 'widgetAjaxCallback'),
+      'callback' => array($this, 'widgetAjaxCallback'),
```

Comment:
I'd like to move forward with this.  With this PR, passing ($this) to the callback function no longer seems to be causing problems.  Its a little concerning to me since I did see it cause problems earlier.  I'm assuming it could have been a core issue with serialization that could have been fixed??  But this is a big problem for us since Modal plugin is broken because of this.  Can we have a discussion today on IRC to talk about this so we can make sure we fix this the correct way and get it committed?